### DISCUSSION TITLE
REQ: Caddy support

### DISCUSSION THREAD

**Discussion by Ondjultomte** [openappsec/openappsec#237] [Feature Requests]
Are there any work beeing done on support for caddy as proxy ?

**Comment by orianelou** [comment]
Hi @Ondjultomte, 

This is currently not on our roadmap, we'll be happy to support any member of the community who will want to add this integration.

**Comment by orianelou** [comment]
If you're interested in this feature, please 👍 or upvote this issue to show your support!
